styles/git-blame.less

```diff
@@ -7,7 +7,6 @@
 
 @git-blame-bg-color: @base-background-color;
 
-atom-text-editor::shadow .git-blame-mount,
```

Comment:
The only reverence I can find related to removing shadow dom is http://flight-manual.atom.io/shadow-dom/sections/removing-shadow-dom-styles/#shadow 

By that section there is no recommendation to add `.editor`. The line that is removed is comes before the current correct line.

If you have an additional reference I will be glad to take a look at that.